job-dsls/jobs/reduced_drools_release.groovy

```diff
@@ -0,0 +1,241 @@
+                        'git checkout -b $baseBranch'
+                }
+            }    
+        }
+        stage ('Clone others') {
+            steps {
+                sshagent(['kie-ci-user-key']) {
+                    sh 'sh droolsjbpm-build-bootstrap/script/release/01_cloneBranches.sh $baseBranch'
+                }    
+            }
+        }
+        // checks if release branch already exists
+        stage ('Check if branch exists') {
+            steps{
+                sshagent(['kie-ci-user-key']) {
+                    dir("${WORKSPACE}" + '/droolsjbpm-build-bootstrap') {
+                        script {
+                            branchExists = sh(script: 'git ls-remote --heads origin ${releaseBranch} | wc -l', returnStdout: true).trim()
+                        }
+                    }
+                }
+            }
+        }
+        stage ('log results') {
+            steps {
+                echo 'branchExists: ' + "$branchExists"
+                script {
+                    if ( "$branchExists" == "1") {
+                        echo "branch exists"
+                    } else {
+                        echo "branch does not exist"
+                    }
+                }
+            }
+        }
+        // if release branches doesn't exist they will be created
+        stage('Create release branches') {
+            when{
+                expression { branchExists == '0'}
+            }
+            steps {
+                sshagent(['kie-ci-user-key']) {
+                    sh 'sh droolsjbpm-build-bootstrap/script/release/02_createReleaseBranches.sh $releaseBranch'
+                }
+            }
+        }
+        // part of the Maven rep will be erased
+        stage ('Remove M2') {
+            steps {
+                sh "sh droolsjbpm-build-bootstrap/script/release/eraseM2.sh $m2Dir"
+            }
+        }
+        stage ('replace repository-list.txt and change branched-7-repository-list.txt') {
+            steps {
+                configFileProvider([configFile(fileId: '1a43573a-318c-426a-bb2b-c9df7fe97a02', targetLocation: 'repository-list.txt', variable: 'REP_LIST')]) {
+                    dir("${WORKSPACE}") {
+                        sh 'cp repository-list.txt droolsjbpm-build-bootstrap/script/repository-list.txt \\n' +
+                           'cat droolsjbpm-build-bootstrap/script/repository-list.txt \\n' +
+                           'rm droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt \\n' +
+                           'touch droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt \\n' +
+                           'echo "optaplanner" >> droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt \\n' +
+                           'cat droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt'
+                    }       
+                }
+            }
+        }         
+        stage ('meet github'){
```

Comment:
the stage name is little bit meaningless ;)
What about Configure github user?

Reply:
OK